music21/analysis/harmonicFunction.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         harmonicFunction.py
+# Purpose:      Mapping between Roman numeral figures and harmonic function labels
+#
+# Authors:      Mark Gotham
+#
+# Copyright:    Copyright © 2022 Michael Scott Cuthbert and the music21 Project
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+
+from typing import Union
+import unittest
+
+from music21 import common
+from music21 import key
+from music21 import roman
+from music21 import scale
+
+from music21 import environment
+_MOD = 'analysis.harmonicFunction'
+environLocal = environment.Environment(_MOD)
```

Comment:
@mscuthbert might be able to speak to this, but I think we can chop these MOD lines going forward.

Reply:
Righto. Let me know.

Reply:
Yeah, I'm just putting it in the environment.Environment('analysis.harmonicFunction') initializer going forward.

Also to note: I'm starting to import typing as t and using t.Optional, etc.  Remembering what has already been imported from typing and what hasn't was beginning to be too much of a mental stress.

Reply:
Thanks @mscuthbert and duly noted x2